tests/unit/tests_high_five_base.cpp

```diff
@@ -129,9 +129,7 @@ BOOST_AUTO_TEST_CASE(HighFiveBasic) {
     BOOST_CHECK_EQUAL(file.getName(), FILE_NAME);
 
     // Create the data space for the dataset.
-    std::vector<size_t> dims;
-    dims.push_back(4);
-    dims.push_back(6);
+    std::vector<size_t> dims = {4,6};
```

Comment:
You can also write dims{4, 6}

Reply:
I've fixed the other uses of `push_back`, as well.